src/Core/SharedKernels/Enumerator/Enumerator/ViewModels/InterviewDetails/EntityWithErrorsViewModel.cs

```diff
@@ -15,20 +16,76 @@ public override void Init(NavigationIdentity entityIdentity, string entityTitle,
             this.navigationState = navigationState;
             this.entityIdentity = entityIdentity;
             this.entityTitle = entityTitle;
+            this.IsError = true;
         }
 
         private NavigationState navigationState;
 
         private string entityTitle;
         public string EntityTitle => this.entityTitle;
+        
+        public bool IsError { get; private set; }
 
         private NavigationIdentity entityIdentity;
 
         public IMvxCommand NavigateToSectionCommand => 
-            new MvxAsyncCommand(async ()=> await this.navigationState.NavigateTo(this.entityIdentity));
+            new MvxAsyncCommand(async ()=> await this.navigationState.NavigateTo(this.entityIdentity), () => entityIdentity != null);
+
+        public static EntityWithErrorsViewModel InitTitle(string title)
+        {
+            var viewModel = new EntityWithErrorsViewModel();
+            viewModel.Init(null, title, null);
+            viewModel.IsError = false;
+            return viewModel;
+        }
+
+        public static EntityWithErrorsViewModel InitError(string title)
+        {
+            var viewModel = new EntityWithErrorsViewModel();
+            viewModel.Init(null, title, null);
+            return viewModel;
+        }
     }
 
     public class EntityWithCommentsViewModel : EntityWithErrorsViewModel
     {
+    }    
+    
+    public class FailCriticalityConditionViewModel : BaseViewModel
+    {
+        public string EntityTitle { get; private set; }
+
+        public void Init(string title)
+        {
+            this.EntityTitle = title;
+        }
     }
+    
+    public class CompleteGroup : MvxObservableCollection<EntityWithErrorsViewModel>
+    {
+        public CompleteGroup()
+        {
+        }
+
+        public CompleteGroup(IEnumerable<EntityWithErrorsViewModel> items) : base(items)
+        {
+        }
+
+        public int AllCount { get; set; }
+        public CompleteGroupContent GroupContent { get; set; }
+            
+        public string Title { get; set; }
+        public bool IsError => GroupContent == CompleteGroupContent.Error && AllCount > 0;

```

Comment:
Unclear logic. Consider do not have objects with AllCount == 0. Use GroupContent Enum value as a logic trigger for consumers.

Reply:
removed